app/controllers/concerns/white_label.rb

```diff
@@ -5,6 +5,8 @@ module WhiteLabel
   include EnterprisesHelper
 
   def hide_ofn_navigation
+    return false unless OpenFoodNetwork::FeatureToggle.enabled?(:white_label)
```

Comment:
Did we say we wanted a feature toggle? I didn't see that on the issue. But if we want it then we should add it to the official list.

Reply:
You're right, I had to search through slack. Here it is for reference: https://openfoodnetwork.slack.com/archives/C01T75H6G0Z/p1679049460596149

> JB will move white labelling behind a dedicated toggle.

Reply:
Done!

Reply:
Thank you for the reference!